script/hassfest/manifest_helper.py

```diff
@@ -1,13 +0,0 @@
-"""Helpers to deal with manifests."""
```

Comment:
Can you please delete the file in a separate PR?

Reply:
Split to #82123.